pkg/tasks/tasks.go

```diff
@@ -104,7 +104,15 @@ func WithHostnameOSAndProbes(t Tasks) Tasks {
 // WithFullInstall with install binaries (using WithBinariesOnly) and
 // orchestrate complete cluster init
 func WithFullInstall(t Tasks) Tasks {
-	return WithBinariesOnly(t).
+	return WithHostnameOSAndProbes(t).append(Tasks{
+		{
+			Fn: func(s *state.State) error {
+				return s.RunTaskOnAllNodes(disableNMCloudSetup, state.RunParallel)
+			},
+			ErrMsg: "failed to disable nm-cloud-setup",
+		},
+	}...).
+		append(WithBinariesOnly(t)...).
```

Comment:
Super tiny nit: this already includes `WithHostnameOSAndProbes`, so the output will look a little bit weird. I propose that  we just do the remaining task, i.e. something like this:

```suggestion
		append(
			Task{Fn: installPrerequisites, ErrMsg: "failed to install prerequisites"},
		).
```